owl/EASE-OBJ.owl

```diff
@@ -1294,35 +2392,329 @@ It is also possible that a Transient transitionsBack to the initial object. An e
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE.owl#FlippingDesign -->
+    <!-- http://www.ease-crc.org/ont/EASE.owl#Dishwasher -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE.owl#Dishwasher">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Appliance"/>
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE.owl#DesignedContainer"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#hasDisposition"/>
+                <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
+                <owl:onClass>
+                    <owl:Class>
+                        <owl:intersectionOf rdf:parseType="Collection">
+                            <rdf:Description rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Enclosing"/>
```

Comment:
A dishwasher is an interesting case where containment of the tableware isn't really exclusionary nor inclusionary. The tableware isn't put inside the dishwasher because it would fly away otherwise (so, not inclusionary) nor is it put in there to protect them from outside stuff, necessarily. They simply are put in the same container where the water jets, who are contained inclusionarily (we don't want them to get out), operate.

Reply:
I don't get what you write here. the tableware is enclosed, so it is included, right?

Reply:
So to say spatially included within the frame of the dishwasher